roles/upgrade/tasks/check-for-old-packages.yml

```diff
@@ -23,13 +23,21 @@
   ansible.builtin.set_fact:
     missing_package_list: "{{ ansible_facts.ansible_local.pre_ripu.unsigned_packages_pre | difference(unsigned_packages_post) }}"
 
-- name: Display warning message if there are any missing non-rhel versioned packages from the pre-upgrade 
+- name: Display warning message if there are any missing non-rhel versioned packages from the pre-upgrade
   ansible.builtin.debug:
-    msg: 
-      - "Warning!! There are non-rhel packages that were removed during the upgrade. Please review the list of missing packages"
+    msg:
+      - "Warning!! There are non-rhel packages that were removed during the upgrade. Please review the list of missingb packages"
       - "{{ missing_package_list }}"
   when: missing_package_list | length >= 1
 
+- name: Save all of the facts from the run for future use/review
+  ansible.builtin.copy:
+    content: "{{ ansible_facts | ansible.builtin.combine({'ansible_local': {}}) }}"
```

Comment:
I don't think we want to save them all just the one fact missing_package_list

Reply:
Updated so it saves as a single local fact called "missing_package_list"